send-surveys/src/main/resources/emailTemplate.mustache

```diff
@@ -0,0 +1,93 @@
+            <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1920 662"><defs><style>.cls-1{fill:#2559a7;}.cls-2{fill:none;stroke:#fff;stroke-miterlimit:10;stroke-width:3px;}.cls-3{fill:#72c8d5;}.cls-4{fill:#00a1b2;}.cls-5{fill:#feb672;}.cls-6{font-size:45px;fill:#fff;font-family:AvenirNext-Bold, Avenir Next;font-weight:700;letter-spacing:0.02em;}</style></defs><title>Illustration Design</title><rect class="cls-1" width="1920" height="662"/><rect class="cls-2" x="104" y="149.5" width="88.54" height="57.95"/><polyline class="cls-2" points="104.2 149.7 148.47 178.68 192.74 149.7"/><path class="cls-2" d="M103.89,207.8c0-.41,33.89-35.78,33.89-35.78l10.38,6.8,10.38-6.8,33.89,35.78" transform="translate(0.32 -0.14)"/><rect class="cls-2" x="1666.64" y="489.11" width="88.54" height="57.95"/><polyline class="cls-2" points="1666.85 489.31 1711.12 518.29 1755.39 489.31"/><path class="cls-2" d="M1666.53,547.41c0-.41,33.89-35.77,33.89-35.77l10.38,6.79,10.38-6.79,33.89,35.77" transform="translate(0.32 -0.14)"/><rect class="cls-2" x="273.64" y="449.11" width="147.51" height="96.55"/><polyline class="cls-2" points="273.98 449.45 347.74 497.72 421.49 449.45"/><path class="cls-2" d="M273.66,546.14c0-.67,56.47-59.6,56.47-59.6l17.29,11.32,17.29-11.32,56.47,59.6" transform="translate(0.32 -0.14)"/><rect class="cls-2" x="1637.16" y="62.81" width="147.51" height="96.55"/><polyline class="cls-2" points="1637.49 63.15 1711.25 111.42 1785.01 63.15"/><path class="cls-2" d="M1637.18,159.84c0-.67,56.46-59.59,56.46-59.59l17.29,11.31,17.3-11.31,56.46,59.59" transform="translate(0.32 -0.14)"/><rect class="cls-2" x="609.19" y="70.98" width="147.51" height="96.55"/><polyline class="cls-2" points="609.53 71.32 683.28 119.6 757.04 71.32"/><path class="cls-2" d="M609.21,168c0-.67,56.47-59.59,56.47-59.59L683,119.74l17.29-11.32L756.72,168" transform="translate(0.32 -0.14)"/><rect class="cls-2" x="1357.78" y="119.76" width="147.51" height="96.55"/><polyline class="cls-2" points="1358.11 120.09 1431.87 168.37 1505.63 120.09"/><path class="cls-2" d="M1357.8,216.79c0-.67,56.46-59.6,56.46-59.6l17.29,11.32,17.3-11.32,56.46,59.6" transform="translate(0.32 -0.14)"/><rect class="cls-3" x="609.5" y="332.5" width="703" height="329.5"/><polygon class="cls-3" points="1312 332 609 332 960 79 1312 332"/><polygon class="cls-4" points="609.34 333 1312.34 333 961.34 586 609.34 333"/><path class="cls-5" d="M1018,257.65H862.33a41.65,41.65,0,0,0-41.65,41.65v82.78a41.64,41.64,0,0,0,41.65,41.65H971l47,35.41V423.73h.09a41.64,41.64,0,0,0,41.64-41.65V299.3A41.64,41.64,0,0,0,1018,257.65Z" transform="translate(0.32 -0.14)"/><text class="cls-6" transform="translate(883.24 348.58)">Hello</text></svg>
+        </div>
+
+        <div class="wrapper">
+          <div class="question">How is your work day?</div>
+          <div class="emoji-row">
+            <div class="face">
+              <img src="grinningFaceBigEyes.png" style="cursor: pointer;" alt="I'm happy";  />
```

Comment:
Throw in those SVGs on these

Reply:
Remove the image files as well...

Reply:
got the img files, the svgs are proving difficult